recipes/poliastro/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.5.0" %}
+
+package:
+  name: poliastro
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/poliastro/poliastro.git
+  git_tag: v{{ version }}
+
+build:
+  number: 1
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - numpy >=1.9
+    - scipy
+    - numba >=0.23
+    - astropy >=1.0
+    - jplephem
+    - matplotlib
+
+test:
+  imports:
+    - poliastro
+    - poliastro.twobody
+	- poliastro.iod
+
+  requires:
+    - pytest
+
+about:
+  home: https://poliastro.github.io/
+  license: MIT License
+  summary: "Python package for Orbital Mechanics"
+
+extra:
+  recipe-maintainers:
+    - Juan Luis Cano Rodríguez
```

Comment:
Here we use GitHub handles. This is important for granting permissions to feedstocks that one creates.